  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_base_link
    pid: {p: 2.0, i: 0.0, d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_1
    pid: {p: 2.0, i: 0.0, d: 0.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_2
    pid: {p: 2.0, i: 0.0, d: 0.0}
